src/snowflake/cli/api/project/schemas/entities/streamlit_entity_model.py

```diff
@@ -31,6 +31,7 @@ class StreamlitEntityModel(EntityModelBase, ExternalAccessBaseModel):
     title: Optional[str] = Field(
         title="Human-readable title for the Streamlit dashboard", default=None
     )
+    comment: Optional[str] = Field(title="Comment for the Streamlit app", default=None)
```

Comment:
You added comment to model but this part is not tested. Could you add test similar to `test_deploy_all_streamlit_files` but for snowflake.yml 2.0?

Reply:
I've added a test case for the StreamlitEntityModel to include the comment.